### PR TITLE
OSDOCS9493: Added configuring certificates to secure routes in your cluster in release notes

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.16+

Issue:https://issues.redhat.com/browse/OSDOCS-9493

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: security/cert_manager_operator/cert-manager-operator-release-notes.adoc [hunk 1]
```diff
@@ -39,6 +39,10 @@ The {cert-manager-operator} now supports the Nokia NetGuard Certificate Manager
 The NCM issuer is validated only with version 1.1.1 and the {cert-manager-operator} version 1.14.0. This version handles tasks such as issuance, renewal, and managing certificates for the API server and ingress controller of {product-title} clusters.
 ====
 
+*Configuring certificates to secure routes in your cluster(Technology Preview)*
```
Comment (adellape) [inline security/cert_manager_operator/cert-manager-operator-release-notes.adoc:42]:
```suggestion
*Configuring certificates to secure routes in your cluster (Technology Preview)*
```

File: security/cert_manager_operator/cert-manager-operator-release-notes.adoc [hunk 2]
```diff
@@ -39,6 +39,10 @@ The {cert-manager-operator} now supports the Nokia NetGuard Certificate Manager
...
+
+With this release, {cert-manager-operator} routes can be configured with third-party certificate management solutions. This enables referencing TLS certificates through secrets, reducing manual management errors and ensuring smooth certificate updates. The OpenShift router promptly serves renewed certificates. For more information, see xref:../../security/cert_manager_operator/cert-manager-securing-routes#cert-manager-configuring-routes_cert-manager-securing-routes[Configuring certificates to secure routes in your cluster].
```
Comment (adellape) [inline security/cert_manager_operator/cert-manager-operator-release-notes.adoc:44]:
```suggestion
With this release, {cert-manager-operator} routes can be configured with third-party certificate management solutions. This enables referencing TLS certificates through secrets, reducing manual management errors and ensuring smooth certificate updates. The {product-title} router promptly serves renewed certificates. For more information, see xref:../.../security/cert_manager_operator/cert-manager-securing-routes.adoc#cert-manager-configuring-routes_cert-manager-securing-routes[Configuring certificates to secure routes in your cluster].
```

**Comment by adellape** [comment]
Related to in-flight https://github.com/openshift/openshift-docs/pull/78500.

File: security/cert_manager_operator/cert-manager-operator-release-notes.adoc [hunk 3]
```diff
@@ -39,6 +39,10 @@ The {cert-manager-operator} now supports the Nokia NetGuard Certificate Manager
 The NCM issuer is validated only with version 1.1.1 and the {cert-manager-operator} version 1.14.0. This version handles tasks such as issuance, renewal, and managing certificates for the API server and ingress controller of {product-title} clusters.
 ====
 
+*Configuring certificates to secure routes in your cluster (Technology Preview)*
+
+With this release, {cert-manager-operator} routes can be configured with third-party certificate management solutions. This enables referencing TLS certificates through secrets, reducing manual management errors and ensuring smooth certificate updates. The {product-title} router promptly serves renewed certificates. For more information, see xref:../.../security/cert_manager_operator/cert-manager-securing-routes.adoc#cert-manager-configuring-routes_cert-manager-securing-routes[Configuring certificates to secure routes in your cluster].
```
Comment (adellape) [inline security/cert_manager_operator/cert-manager-operator-release-notes.adoc:44]:
```suggestion
With this release, {cert-manager-operator} routes can be configured with third-party certificate management solutions. This enables referencing TLS certificates through secrets, reducing manual management errors, and ensuring smooth certificate updates. The {product-title} router promptly serves renewed certificates. For more information, see xref:../.../security/cert_manager_operator/cert-manager-securing-routes.adoc#cert-manager-configuring-routes_cert-manager-securing-routes[Configuring certificates to secure routes in your cluster].
```

File: security/cert_manager_operator/cert-manager-operator-release-notes.adoc [hunk 4]
```diff
@@ -39,6 +39,10 @@ The {cert-manager-operator} now supports the Nokia NetGuard Certificate Manager
 The NCM issuer is validated only with version 1.1.1 and the {cert-manager-operator} version 1.14.0. This version handles tasks such as issuance, renewal, and managing certificates for the API server and ingress controller of {product-title} clusters.
 ====
 
+*Configuring certificates to secure routes in your cluster (Technology Preview)*
+
+With this release, you can manage certificate referenced in `route` resources by using the {cert-manager-operator}. For more information, see xref:../.../security/cert_manager_operator/cert-manager-securing-routes.adoc#cert-manager-securing-routes[Securing routes with the cert-manager Operator for Red Hat OpenShift].
```
Comment (lunarwhite) [inline security/cert_manager_operator/cert-manager-operator-release-notes.adoc:44]: How about
```suggestion
With this release, you can manage certificate referenced in `route` resources by using the {cert-manager-operator}. For more information, see xref:../../security/cert_manager_operator/cert-manager-securing-routes.adoc#cert-manager-securing-routes[Securing routes with the {cert-manager-operator}].
```

File: security/cert_manager_operator/cert-manager-operator-release-notes.adoc [hunk 5]
```diff
@@ -39,6 +39,10 @@ The {cert-manager-operator} now supports the Nokia NetGuard Certificate Manager
 The NCM issuer is validated only with version 1.1.1 and the {cert-manager-operator} version 1.14.0. This version handles tasks such as issuance, renewal, and managing certificates for the API server and ingress controller of {product-title} clusters.
 ====
 
+*Configuring certificates to secure routes in your cluster (Technology Preview)*
```
Comment (lunarwhite) [inline security/cert_manager_operator/cert-manager-operator-release-notes.adoc:42]:
```suggestion
*Securing routes with cert-manager managed certificates (Technology Preview)*
```

**Review by lunarwhite — COMMENTED** [review]
Thank you! Could you also place this part between the FIPS part and the NCM part? (Just to align the order given by Anjali)

File: security/cert_manager_operator/cert-manager-operator-release-notes.adoc (hunk 4 above)
Comment (subhtk) [reply security/cert_manager_operator/cert-manager-operator-release-notes.adoc:44]: Might have some rendering issues. Will check.

File: security/cert_manager_operator/cert-manager-operator-release-notes.adoc [hunk 6]
```diff
@@ -30,6 +30,10 @@ Version `1.14.0` of the {cert-manager-operator} is based on the upstream cert-ma
 
 With this release, FIPS mode is now automatically enabled for {cert-manager-operator}. When installed on an {product-title} cluster in FIPS mode, {cert-manager-operator} ensures compatibility without affecting the cluster's FIPS support status.
 
+*Securing routes with cert-manager managed certificates (Technology Preview)*
+
+With this release, you can manage certificate referenced in `route` resources by using the {cert-manager-operator}. For more information, see xref:../../security/cert_manager_operator/cert-manager-securing-routes.adoc#cert-manager-securing-routes[Securing routes with the {cert-manager-operator}].
```
Comment (adellape) [inline security/cert_manager_operator/cert-manager-operator-release-notes.adoc:35]:
```suggestion
With this release, you can manage certificates referenced in `Route` resources by using the {cert-manager-operator}. For more information, see xref:../../security/cert_manager_operator/cert-manager-securing-routes.adoc#cert-manager-securing-routes[Securing routes with the {cert-manager-operator}].
```

**Review by adellape — APPROVED** [review]

**Pull request merged**